<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditImageBase</class>
 <widget class="QWidget" name="EditImageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="verb, to modify selected image attributes">Edit Image Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelWidth">
     <property name="text">
      <string comment="noun, width of the image in pixels">Width:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinboxWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KIntSpinBox" name="spinboxWidth">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="KIntSpinBox" name="spinboxHeight">
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Image Title:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>txtTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelHeight">
     <property name="text">
      <string comment="noun, height of the image in pixels">Height:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinboxHeight</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="KLineEdit" name="txtTitle">
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelAltText">
     <property name="text">
      <string comment="the alternative text which is displayed to the user, if the image can not be loaded.">Alternative text:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>txtAltText</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="KLineEdit" name="txtAltText">
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelLink">
     <property name="text">
      <string>Image links to:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>txtLink</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="KLineEdit" name="txtLink">
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="KSeparator" name="kseparator"/>
   </item>
   <item row="0" column="2">
    <widget class="KPushButton" name="btnKeepRatio">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Image aspect ratio">Keep Ratio</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTitle_2">
     <property name="text">
      <string extracomment="Image Alignment">Alignment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>txtTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="KComboBox" name="alignment">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string extracomment="Alignment">Not Set</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="Alignment">Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="Alignment">Left</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
